### PR TITLE
Install infra and nova operator for dataplane kuttl tests

### PR DESCRIPTION
**Pull request by rabi**
We need the crds, though the extent of testing we do is fine without the controller-manager.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1380,7 +1380,7 @@ dataplane_kuttl_prep: dataplane_kuttl_cleanup
 
 .PHONY: dataplane_kuttl
 # dataplane must come before dataplane_kuttl_prep since dataplane creates the CRDs
-dataplane_kuttl: input dataplane ansibleee namespace dataplane_kuttl_prep operator_namespace ## runs kuttl tests for the openstack-dataplane operator. Installs openstack-dataplane operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
+dataplane_kuttl: input dataplane ansibleee infra nova namespace dataplane_kuttl_prep operator_namespace ## runs kuttl tests for the openstack-dataplane operator. Installs openstack-dataplane operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
 	$(eval $(call vars,$@,dataplane))
```
Comment (kajinamit) [inline Makefile:1384]: infra target and nova target do not include the wait step now. So you probably have to add the following lines to make sure these operators are installed.

```
$(eval $(call vars,$@,infra))
make wait
$(eval $(call vars,$@,nova))
make wait
```

**Comment by dprince** [comment]
@rabi with the latest re-architecture I think NovaExternalCompute is going to go away. We can however merge this now, and then remove that dependency at a later time.

**Comment by rabi** [comment]
@dprince dataplane kuttl jobs are broken because we changed recently to not create all crds before hand. This is required atm to fix those jobs.

**Comment by rabi** [comment]
recheck

**Comment by rabi** [comment]
recheck

**Comment by rabi** [comment]
recheck

**Comment by rabi** [comment]
recheck

**Review by fao89 — APPROVED** [review]

**Pull request merged**